curriculum/challenges/arabic/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/6569f70a66ccdc42097ca051.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-64
 
 # --description--
 
-If still there are empty cells, create a loop in the `else` block that iterates over numbers from `1` to `9` (inclusive).
+If still there are empty cells, create a loop in the `else` block. Use `guess` to iterate over numbers from `1` to `9` (inclusive).
```

Comment:
Please do not make changes to non english files.

Please revert this back to the original

Reply:
Understood. That was an accident.